packages/FormElement/src/FormElement.js

```diff
@@ -71,85 +72,73 @@ function FormElement(props) {
     ...moreProps
   } = props;
 
-  const extratedChildren = extractChildren(children, [
+  const extractedChildren = extractChildren(children, [
     "FormElement.Description",
     "FormElement.Error",
     "FormElement.Help",
+    "FormElement.ExtraPanel",
   ]);
   const ariaDescriptionId = React.useRef(uuidv4()).current;
-  const hasError = !!extratedChildren["FormElement.Error"] && !!extratedChildren["FormElement.Error"].props.children;
+  const hasError = !!extractedChildren["FormElement.Error"] && !!extractedChildren["FormElement.Error"].props.children;
   const uniqueInputId = React.useRef(uuidv4()).current;
   const inputId = isNil(id) || id === "" ? uniqueInputId : id;
-  let isFooterInserted = false;
 
   function renderFooter() {
-    if (isFooterInserted) return;
-
-    isFooterInserted = true;
-
     if (hasError) {
-      return extratedChildren["FormElement.Error"];
+      return extractedChildren["FormElement.Error"];
     }
 
-    if (extratedChildren["FormElement.Description"]) {
-      return React.cloneElement(extratedChildren["FormElement.Description"], {
+    if (extractedChildren["FormElement.Description"]) {
+      return React.cloneElement(extractedChildren["FormElement.Description"], {
         ariaDescriptionId,
       });
     }
 
     return null;
   }
 
+  const childAttributes = {
+    className: "form-element--child",
+    "aria-describedby": ariaDescriptionId,
```

Comment:
The new component will also need an id that can be included here.  The form control needs to have a list of element ids that describe it, including the description, the error, AND the new component.